src/components/Table/Table.module.css

```diff
@@ -141,15 +141,46 @@
   }
 }
 
+/**
+* Table header cell
+* 1) When column is sortable, add an icon to the th after the title
+* 2) On hover over the th, the sort icon animates to opacity 1
+* 3) Cursor:pointer is used on the sort icon only
+* 
+*/
 .table__header-cell {
   @mixin eds-theme-typography-label-sm;
   padding: var(--eds-size-1) var(--eds-size-1);
   text-align: left;
 
-  .table__body & {
-    padding: var(--eds-size-2) var(--eds-size-1);
-    @mixin eds-theme-typography-body-text-xs-bold;
+  /* 1 */
+  .table__header-cell--sort {
```

Comment:
Change this to `table__header-cell-sort`. Although this will likely become the button around the text and the icon.

Reply:
Done